scripts/install-deps.sh

```diff
@@ -0,0 +1,4 @@
+#!/bin/bash -e
+
+curl -sSfL "https://raw.githubusercontent.com/Kong/go-atc-router/trunk/build-deps.sh" \
```

Comment:
Have we thought about local development requirements? What would the process look like if we needed to make updates to `go-atc-router` & need to have multiple versions in the mix?